packages/frontend/src/components/TableConfigPanel/index.tsx

```diff
@@ -15,7 +15,7 @@ export const TableConfigPanel: React.FC = () => {
         <Popover2
             content={
                 <ConfigWrapper>
-                    <ConfigLabel>Show table names</ConfigLabel>
+                    <Label>Show table names</Label>
```

Comment:
From looking at the other panels we seem to use a custom `p`. Maybe we should use it here as well.

```
export const SectionTitle = styled.p`
    color: ${Colors.DARK_GRAY1};
    font-weight: 600;
    margin-bottom: 0.286em;
`;
```